src/server/service/kv.rs

```diff
@@ -1617,6 +1624,18 @@ txn_command_future!(future_check_txn_status, CheckTxnStatusRequest, CheckTxnStat
             Err(e) => resp.set_error(extract_key_error(&e)),
         }
 });
+txn_command_future!(future_check_secondary_locks, CheckSecondaryLocksRequest, CheckSecondaryLocksResponse, (status, resp) {
+    // All possible errors should be extracted by `extract_region_error`.
```

Comment:
Does it cover all errors including errors returned by `MvccReader`?

Reply:
Ah, I thought the errors come from the engine and they were covered. Now I find bad write record format also has an error.  So a key error is necessary... https://github.com/pingcap/kvproto/pull/657/files